### PR TITLE
User Webhook Message Links

### LINKED ISSUE #278: Use Webhooks for message links
Use a webhook with the Users name & maybe profile picture of the User that sent the initial message

### PR DESCRIPTION
**Pull request by DynxstyGIT**
Closes #278

### CODE REVIEW

**Comment by DynxstyGIT** [comment]
This would work best with buttons imo. However, Minn's Webhook Lib doesn't support them (yet: https://github.com/MinnDevelopment/discord-webhooks/issues/46). 
Some dude actually made a PR for this, but never finished and just closed their PR at some point (https://github.com/MinnDevelopment/discord-webhooks/pull/56).

Lemme see if I can get his PR to work soonTm

**Comment by DynxstyGIT** [comment]
I fixed the issues myself and made this work now 👍

File: build.gradle.kts [hunk 1]
```diff
@@ -41,7 +41,7 @@ dependencies {
     implementation("org.quartz-scheduler:quartz:2.3.2")
     
     // Webhooks
-    implementation("club.minnced:discord-webhooks:0.8.0")
+    implementation("com.github.DynxstyGIT:discord-webhooks:74301a46a0")
```
Comment (danthe1st) [inline build.gradle.kts:44]: Why is that required?

Comment (MoonTM-GIT) [reply build.gradle.kts:44]: Look at his comments, he added button support

**Review by MoonTM-GIT — APPROVED** [review]

**Review by danthe1st — APPROVED** [review]
